Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
Traceback (most recent call last):
  File "/home/ubuntu/automated-prescription-system/prescription_automation.py", line 5, in <module>
    from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'
